server/controllers/groups.js

```diff
@@ -69,10 +69,13 @@ export const getUsers = (req, res, next) => {
   const tiers = _.groupBy(req.group.tiers, 'name');
 
   const _isActive = (backer) => {
-    if (tiers[backer.tier] && tiers[backer.tier].interval === 'monthly' && now.diff(moment(backer.lastDonation), 'days') > 31)
-      return false
-    else
-      return true;
+    if (tiers[backer.tier]) {
+      if (tiers[backer.tier][0].interval === 'monthly' && now.diff(moment(backer.lastDonation), 'days') > 31)
+        return false;
+      if (now.diff(moment(backer.lastDonation), 'days') > 365)
```

Comment:
Doesn't this mean it's better for me to make a one-time donation to stay up there for a year?

I think we should cap one-time donations at 31 days as well to be fair to recurring donors.